roles/pulp_workers/README.md

```diff
@@ -3,12 +3,12 @@ pulp_workers
 
 Install, configure, and set the state of pulp workers.
 
-Configurable Variables:
+Configurable Variables
```

Comment:
MD works either way, but visually I think it's nicer when the underline part is the same length as the title it underlines. This could be checked in all areas of this PR.